mlflow/projects/__init__.py

```diff
@@ -43,36 +43,35 @@ def _run(uri, entry_point="main", version=None, parameters=None, experiment_id=N
     Returns a `SubmittedRun` corresponding to the project run.
     """
     exp_id = experiment_id or tracking._get_experiment_id()
+    params = parameters or {}
```

Comment:
Maybe we can overwrite `parameters` -- otherwise it's pretty likely that we will accidentally use `parameters` as opposed to `params` later on in the function.